17_foundations.ipynb

```diff
@@ -1066,7 +1066,7 @@
     "\n",
     "```\n",
     "Image  (3d tensor): 256 x 256 x 3\n",
-    "Scale  (1d tensor):  (1)   (1)  3\n",
+    "Scale  (2d tensor):  (1)   (1)  3\n",
```

Comment:
Why did you change this? It appears to be a 1d tensor AFAICT?

Reply:
I am not sure if this is consistent with the description above:
> Arrays do not need to have the same number of dimensions. For example, if you have a 256×256×3 array of RGB values, and you want to scale each color in the image by a different value, you can multiply the image by a **one-dimensional array with three values.**

Maybe I am mistaken, but this caught my attention.

Reply:
It is a one-dimensional tensor, as it says, so it should say "1d" in the summary, not "2d".

Reply:
Right, I fixed the wrong tensor and this confused me. Sorry.